src/igvfd/tests/data/inserts/human_donor.json

```diff
@@ -0,0 +1,126 @@
+[
+    {
+        "uuid": "9050f480-1d11-4d2a-b6f2-5979c2af4fd6",
+        "lab": "j-michael-cherry",
+        "award": "HG012012",
+        "aliases": [
+            "j-michael-cherry:alias_human_donor1"
+        ],
+        "status": "released",
+        "submitter_comment": "The submitter had very little to say.",
+        "documents": [],
+        "notes": "This is a note.",
+        "url": "https://www.genome.gov/Funded-Programs-Projects/Impact-of-Genomic-Variation-on-Function-Consortium",
+        "health_status_history": [
+            {
+                "health_description": "Running fever 103F, feeling tired",
+                "date_start": "2022-01-04",
+                "date_end": "2022-01-10"
+            },
+            {
+                "health_description": "Running fever 103F, feeling tired",
+                "date_start": "2021-12-03",
+                "date_end": "2022-01-10"
+            }
+        ],
+        "ethnicity": [
+            "Hispanic"
+        ],
+        "genetic_modifications": [
```

Comment:
does it make sense to have a genetic modification on a human.  In general, our notes said that we were not going to include this on donor.

Reply:
drop genetic modification

Reply:
Done